Base/usr/share/man/man1/shuf.md

```diff
@@ -0,0 +1,19 @@
+## Name
```

Comment:
I've been using that script to generate the man pages I've been adding, then adding descriptions in the `Name` section and making other small format tweaks. 

I imagine that we could make the CI script that checks the man pages more intelligent in some way, so that it can verify things like the `Options` section all correct, while ignoring things like `Examples`, which need to be manually added.

Reply:
Ah, then someone (in the future, not in this PR) should probably add a way to make the examples auto-generatable. I dislike that we have two separate places that document the same program, but that can't be fixed here. Thanks for the response!